src/tensors/abstracttensor.jl

```diff
@@ -38,7 +38,7 @@ spacetype(::Type{<:AbstractTensorMap{<:Any,S}}) where {S} = S
 
 Return the type of sector `I` of a tensor.
 """
-sectortype(::Type{TT}) where {TT<:AbstractTensorMap} = sectortype(spacetype(TT))
+sectortype(::Type{<:AbstractTensorMap{<:Any,S}}) where {S} = sectortype(S)
```

Comment:
Is there a specific reason for this change? I tens to avoid accessing type parameters as much as possible, as this is very annoying to change if ever the order changes, hence the previous definition in terms of a function instead of accessing the type parameters. It does not really matter too much, just curious if there was a specific benefit

Reply:
No probably not, I'll revert this.